src/services/services.ts

```diff
@@ -1725,7 +1725,7 @@ namespace ts {
             const sourceFile = syntaxTreeCache.getCurrentSourceFile(fileName);
 
             // Check if in a context where we don't want to perform any insertion
-            if (isInString(sourceFile, position) || isInComment(sourceFile, position)) {

```

Comment:
c# is smart, otherwise too outside comments, it only completes when brackets are placed in an expected location